SetDefaultSinkId/explainer.md

```diff
@@ -0,0 +1,185 @@
+# Set Default Audio Output Device
+
+Author: [Sunggook Chue] [Ravikiran Ramachandra] [Steve Becker] [Andy Luhrs]
+
+## Status of this Document
+This document is a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+
+This document status: Active
+Expected venue: [WebRTC WG](https://www.w3.org/groups/wg/webrtc)
+Current version: this document
+
+## Introduction
+
+Setting Audio Output Device In Frames
```

Comment:
This reads like a subheading but isn't formatted like one. Either format it that way or turn it into one or more sentences that provide a higher-level introduction up front, e.g. something like, "Sites that embed third-party content they don't fully control would like the ability to influence which device audio gets routed for their entire page in response to a user stating they have a preferred device. To enable that, this proposal allows controlling the default audio output device for the entire page, including subframes."

Reply:
I will use your sentence as is and see how it looks like. thanks.